src/common/components/Input/Icons/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './ViewOn';
+export * from './ViewOff';
```

Comment:
Related to the previous comments, here we would then export:

```suggestion
export ViewOn from './ViewOn.svg';
export ViewOff from './ViewOff.svg';
```